ansible/roles/openstack-ansible-os_triliovault_wlm/tasks/uninstall_wlm_pckgs.yml

```diff
@@ -39,23 +39,12 @@
   when: >
     (ansible_distribution_major_version>="20.04" and ansible_distribution=="Ubuntu")
 
-- name: Uninstall workloadmgr pckg
+- name: Uninstall workloadmgr packages
   package:
-    name: workloadmgr
+    name: 
+      - workloadmgr
```

Comment:
mention other pkgs too, wlmclient, contegoclient and s3fuseplugin

Reply:
Added the required changes
It will uninstall the below packages

- python3-s3-fuse-plugin when platform is CentOS/RHEL
- python3-s3fuse-plugin when platform is Ubuntu/Debian